metabase/metabase
# static chart legend

Epic: https://github.com/metabase/metabase/issues/24759

## Changes

Replaces the old two-column static chart legend with a row legend at the top.

<img width="647" alt="Screen Shot 2022-10-26 at 3 14 01 PM" src="https://user-images.githubusercontent.com/14301985/198012220-0bc0c665-064e-4498-8b71-671b7abc2492.png">

## How to verify

- Open the combo-chart in the Storybook
- Send a test subscription of a chart that should have a legend
